src/connector/src/sink/redis.rs

```diff
@@ -24,23 +24,21 @@ use risingwave_common::catalog::Schema;
 use serde_derive::{Deserialize, Serialize};
 use serde_with::serde_as;
 
+use super::catalog::SinkFormatDesc;
 use super::formatter::SinkFormatterImpl;
 use super::writer::FormattedSink;
-use super::{SinkError, SinkParam, SINK_TYPE_APPEND_ONLY, SINK_TYPE_OPTION, SINK_TYPE_UPSERT};
+use super::{SinkError, SinkParam};
 use crate::dispatch_sink_formatter_impl;
 use crate::sink::writer::{LogSinkerOf, SinkWriterExt};
 use crate::sink::{DummySinkCommitCoordinator, Result, Sink, SinkWriter, SinkWriterParam};
 
 pub const REDIS_SINK: &str = "redis";
-
+pub const REDIS_KEY_FORMAT: &str = "redis_key_format";
+pub const REDIS_VALUE_FORMAT: &str = "redis_value_format";
```

Comment:
The `template` encoder may be used for other sinks in the future. What about just naming these `key.format` and `value.format`?

FYI we have `key.message` and `message` for avro / proto in source today.